### PR TITLE
POL-130 - Master Org Scheduled Report with Currency Conversion

### PR DESCRIPTION
**Pull request by srpomeroy**
### Description

A policy that spans multiple organizations and combines costs under a single currency.

### Issues Resolved

N/A

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt [hunk 1]
```diff
@@ -0,0 +1,1026 @@
...
+short_description "This policy allows you to set up cross organization scheduled reports that will provide summaries of cloud cost across all organizations you have access to, delivered to any email addresses you specify. The policy will report the following:
+Chart of the previous 6 months of utilization based on [category](https://docs.rightscale.com/optima/reference/rightscale_dimensions.html#category).  
+Daily average cost across the last week and last month.  
+Total cost during previous full week (Monday-Sunday) and previous full month.  
+Total cost during current (incomplete) week and month.  
+We recommend running this policy on a weekly cadence and applying it to your master account.
+_Note 1: The last 3 days of data in the current week or month will contain incomplete data._  
+_Note 2: The account you apply the policy to is unimportant as Optima metrics are scoped to the Org._  
+_Note 3: Exchange rates are calculated at execution time using [https://exchangeratesapi.io/](https://exchangeratesapi.io/)._  
+See [README](https://github.com/rightscale/policy_templates/tree/master/msp/cost/master_org_cost_policy) for more details"
```
Comment (cdwilhelm) [inline msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt]: The link to this readme is not accurate.

File: msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt [hunk 2]
```diff
@@ -0,0 +1,1026 @@
...
+
+| Range | Daily Average | Previous | Current (incomplete) |
+| ----- | ------------- | -------- | -------------------- |
+{{ range data.reportData -}}
+  | {{ .range }} | {{ .avg }} | {{ .previous}} | {{ .current }} |
+{{ end -}}  
+  
+### For more detailed cost information, visit [Optima](https://analytics.rightscale.com/orgs/{{ rs_org_id }}/dashboard).
+  
+For more information on this report, please view the [README](https://github.com/rightscale/policy_templates/tree/master/cost/master_org_cost_policy_currency).<br>
```
Comment (cdwilhelm) [inline msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt]: This isn't the correct path for the readme

File: msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt (hunk 1 above)
Comment (srpomeroy) [reply msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt]: @cdwilhelm resolved

File: msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt (hunk 2 above)
Comment (srpomeroy) [reply msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt]: @cdwilhelm resolved

**Pull request merged**